src/ui/components/shared/Login/Login.tsx

```diff
@@ -175,21 +175,33 @@ function AuthError({ error }: { error: any }) {
     return null;
   }
 
-  if (message === "Invalid state") {
-    // This is usually caused by waiting too long to go through the auth process
-    // and can be fixed by trying again.
-    message = "Your login session expired. Please try logging in again.";
-  } else {
-    // We want to capture any other error so we can investigate further.
+  if (message !== "INVALID_STATE" && message !== "Invalid state") {
```

Comment:
Where do these strings come from (e.g. "INVALID_STATE")

Reply:
This comes from our Auth0 Rule, something we define, except "Invalid state". Ryan and I plan to implement enum-like errors before merging this PR, moving away from string matching on human-readable messages.

Reply:
Sorry I don't know how I seem to have added the same comment twice 😆